insurance/monitoring/monitoring-ui.md

```diff
@@ -191,15 +191,50 @@
  - Select `Submit` button on the bottom right
  - This will show that the Alert has been set
  
-### 6. Upload threshold file, 
-- From model monitor actions, click on Upload thresholds. 
-- Download the threshold file [thresholds.json](https://github.com/oneconvergence/dkube-examples/blob/monitoring/insurance_cloudevents/thresholds.json) and upload.
-
-### 7. Start Monitor.
-Click on Start for the specific monitor on Modelmonitor dashboard.
-   - Modelmonitor can only be started in 'ready' state.
-   - It can be stopped anytime. Previous data will not be erased.
-
-### 8. Data Generation
- - Go to the IDE and open `workspace/insurance/insurance/monitoring/data_generation.ipynb`
- - Run all cells.
+### Upload Threshold File
+ 
+ It is possible to set a range of alert thresholds through a configuration file uploaded to DKube.  This section explains how to upload a threshold file.
```

Comment:
Note: thresholds file doesn't set alerts. It is only used for visuals (green, yellow, red) on UI. One can set alerts later either based on these thresholds or completely new thresholds.

Reply:
I have added the note to my fork.  It will get merged during the next commit.